filter/ArticleMedraXmlFilter.inc.php

```diff
@@ -331,6 +405,21 @@ function createContentItemNode($doc, $issue, $article, $galley, $objectLocalePre
 			}
 
 		}
+		//Adding Citation List (unstructured) on the content
+		$citationDao = DAORegistry::getDAO('CitationDAO'); /* @var $citationDao CitationDAO */
+		$parsedCitations = $citationDao->getBySubmissionId($article->getId())->toArray();
+		$citationListNode = $doc->createElementNS($deployment->getNamespace(), 'cl:CitationList');
+		$citationListNode->setAttributeNS('http://www.w3.org/2000/xmlns/' ,'xmlns:cl', 'http://www.medra.org/DOIMetadata/2.0/Citations');
+		foreach($parsedCitations as $citation) {
```

Comment:
Could also here a separate function be created to add the citations?

Reply:
I put the part of code for building the metadata of the Citations in a separate function.

Reply:
Could you please let me know if we can go ahead with this new implementation? Thanks ans regards